src/earn/hooks.ts

```diff
@@ -17,52 +17,19 @@ export function useAaveRewardsInfoAndPrepareTransactions({
   poolTokenId,
```

Comment:
can we rename this function? Maybe even move to prepareTransactions, similar to how the deposit/supply one works?

Reply:
Rename, fair. I'll do the refactor to be more like deposit/supply in the next PR